gubernator/static/pr.js

```diff
@@ -0,0 +1,82 @@
+
+var upArrow = "\u25b2";
+var downArrow = "\u25bc";
+
+function sortRows(rows, col, colName, dir) {
+  // customize sorting functions for different columns
+  var extract = function(el) { return el.innerText.replace(/^\s+|\s+$/g, ''); }
```

Comment:
Does the /g really do stuff in js?

Reply:
Yes, it matches multiple times instead of a single time.